ConsoleApp/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace ConsoleApp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            /*
+            Create a program that will start with declaration of two constants (X and Y) and will count the sum of all numbers between these constants. 
+            If they are equal then sum should be one of them
+
+            Example:
+            X=10
+            Y=12
+            Sum=10+11+12=33
+            */
+            try
+            {
+                Console.WriteLine("Enter the x: ");
+                int x = Convert.ToInt32(Console.ReadLine());
```

Comment:
it will raise an exception if I enter not a number, that's not the case, we've discussed to handle it with asking for user to enter number again